<!-- MES Client launch file -->

<launch>
	<group ns="RC_MES_Client">
		<node pkg="rc_mes_client" name="rc_mes_client" type="rc_mes_client">
			<param name="hmiConsole" value="/rcHMI/console"/>
			<param name="mesPub" value="/rcMESClient/msgFromServer"/>
			<param name="mesSub" value="/rcMESClient/msgToServer"/>
			<param name="server_ip" value="10.115.253.233"/>
			<param name="server_port" value="21240"/>
		</node>
	</group>
</launch> 
